### ISSUE TITLE
Option to continue building on failure when using --recipe-dir

### ISSUE THREAD

**Issue by KGB99** [prefix-dev/rattler-build#1024]
When using the `--recipe-dir` flag it would be useful to be able to continue building the rest of the recipes even if one of them fails. I've begun with some changes in #1023 however I have not entirely grasped how to add the flag to the CLI yet.

**Comment by wolfv** [comment]
Yes, agreed, that would be nice! I'll check your changes asap (was on vacation for the past two weeks).

**Comment by wolfv** [comment]
Done in #1626 – thanks for your prior work @kerimbirgi!
